src/components/screens/PostsFeed/PostsFeed.js

```diff
@@ -46,40 +45,49 @@ class PostsFeed extends PureComponent<Props, State> {
 
   render() {
     let filter;
-    const {posts} = this.props;
-    const {searchName, toggleSearch} = this.state;
-    (searchName !== '')
-      ? filter = this.filtering(posts, searchName)
-      : filter = posts;
-    return (<View style={styles.container}>
-      <View style={styles.header}>
-        <ButtonIcon iconName="search" setToggle={this._setToggle}/>
-        <HeaderTitle text="FEED"/>
-        <ButtonIcon iconName="plus"/>
-      </View>
-      {
-        toggleSearch
-          ? (<TextInput style={styles.textInput}
+    const { posts } = this.props;
+    const { searchName, toggleSearch } = this.state;
+    searchName !== ""
+      ? (filter = this.filtering(posts, searchName))
+      : (filter = posts);
+    return (
+      <View style={styles.container}>
+        <View style={styles.header}>
+          <ButtonIcon iconName="search" setToggle={this._setToggle} />
+          <HeaderTitle text="FEED" />
+          <ButtonIcon iconName="plus" />
+        </View>
+        {toggleSearch ? (
+          <TextInput
+            style={styles.textInput}
             placeholder="Search"
             value={searchName}
-            onChangeText={searchName => this.setState({searchName})}/>)
-          : (<View/>)
-      }
-      <FlatList removeClippedSubviews={false}
-        disableVirtualization="disableVirtualization"
-        showsVerticalScrollIndicator={false}
-        initialNumToRender={15}
-        maxToRenderPerBatch={10}
-        data={filter}
-        keyExtractor={item => item._id}
-        renderItem={({item}) => <View>
-          <UserPost
-            userName={item.userName}
-            uri={{ uri: item.uri }}
-            uriPhoto={{ uri: item.uriPhoto }}/>
-        </View>}/>
-    </View>);
-  };
+            onChangeText={searchName => this.setState({ searchName })}
```

Comment:
Don't bind functions in render, it will cause recreation of the function, move it to class function
```
onChangeText = (text: string) => {
  this.setState({ searchName })
}
```
and in render
```
<TextInput
  ...
  onChangeText={this.onChangeText}
```